hudi-cli/src/main/java/org/apache/hudi/cli/commands/CompactionCommand.java

```diff
@@ -381,20 +382,21 @@ private HoodieCompactionPlan readCompactionPlanForArchivedTimeline(HoodieArchive
    */
   private HoodieCompactionPlan readCompactionPlanForActiveTimeline(HoodieActiveTimeline activeTimeline,
                                                                    HoodieInstant instant) {
+    InstantGenerator instantFactory = HoodieCLI.getTableMetaClient().getInstantGenerator();
```

Comment:
I see many places where the class name is renamed to `xxxGenerator` but the variable name is still `xxxFactory`, can we fix that?

Reply:
Fixed.

Reply:
To fix this, I mean renaming `instantFactory` to `instantGenerator`